modules/configuring-a-custom-notebook.adoc

```diff
@@ -26,15 +26,16 @@ endif::[]
 .Procedure
 . From the {productname-short} dashboard, click *Settings* -> *Notebook images*.
 +
-The *Notebook image settings* page appears. Previously imported notebook images are displayed. To enable or disable a previously imported notebook image, on the row containing the relevant notebook image, click the toggle in the *Enable* column.
+The *Notebook image settings* page appears. Previously imported notebook images are displayed. To enable or disable a previously imported notebook image, on the row containing the relevant notebook image, click the toggle in the *Enable* column. Click *Create profile* on the row containing the relevant notebook image to create an accelerator profile as the image's recommended accelerator.  
 . Click *Import new image*. Alternatively, if no previously imported images were found, click *Import image*.
 +
 The *Import Notebook images* dialog appears.
 . In the *Image location* field, enter the URL of the repository containing the notebook image. For example: `quay.io/my-repo/my-image:tag`, `quay.io/my-repo/my-image@sha256:xxxxxxxxxxxxx`, or
 `docker.io/my-repo/my-image:tag`.
 
 . In the *Name* field, enter an appropriate name for the notebook image.
-. In the *Description* field, enter an appropriate description for the notebook image.
+. Optional: In the *Description* field, enter a description for the notebook image.
+. Optional: From the *Accelerator identifier* list, select an identifier to set its accelerator as recommended with the notebook image.
```

Comment:
This is not always true. 
If the image has only one Accelerator identifier, there is no dropdown (list) and the values should be fixed with that Accelerator identifier.

Reply:
Fixed.